lucca.conf

```diff
@@ -24,7 +24,7 @@ badge_override_interval=10
 bind_address=0.0.0.0
 bind_port=3000
 
-pkcs_cert_file=test/cert/certificate.p12
+pkcs_cert_file=path/to/cert.p12
```

Comment:
Will this break being able to set up the app right out of the box? Just wondering if there is an issue using the current certificate in the test folder (expired cert?)

Reply:
Yes, it will. This is intentional. Having it "just work" immediately upon download discourages people from figuring out how to configure the service properly, meaning they are more likely to use the cert provided for testing (one one that was preciously linked).
Encouraging people to use this cert is undesirable because while it technically works, the private key is trivially easy to find online, meaning the security provided is essentially a complete illusion. We want people not to use this cert without realizing the mistake they're making. They should be forced to source their own cert, even if its self-signed, so that they don't mistakenly assume their traffic is secure when it is not.

Reply:
For what it's worth, this kind of "doesn't work until you fix it" default config parameter is pretty common practice for industry-standard open source software projects that require the user to provide a certificate.